src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java

```diff
@@ -687,23 +687,19 @@ public void setAllowEmptyCatches(boolean allow) {
     public void visitToken(DetailAST ast) {
         final int currentType = ast.getType();
         if (!isNotRelevantSituation(ast, currentType)) {
-            final String line = getLine(ast.getLineNo() - 1);
+            final int[] line = getLineCodePoints(ast.getLineNo() - 1);
             final int before = ast.getColumnNo() - 1;
             final int after = ast.getColumnNo() + ast.getText().length();
-            final int[] codePoints = line.codePoints().toArray();
 
-            if (before >= 0) {
-                final char prevChar = Character.toChars(codePoints[before])[0];
-                if (shouldCheckSeparationFromPreviousToken(ast)
-                        && !Character.isWhitespace(prevChar)) {
-                    log(ast, MSG_WS_NOT_PRECEDED, ast.getText());
-                }
+            if (before >= 0 && shouldCheckSeparationFromPreviousToken(ast)
+                        && !CommonUtil.isCodePointWhitespace(line, before)) {
+                log(ast, MSG_WS_NOT_PRECEDED, ast.getText());
             }
 
-            if (after < codePoints.length) {
-                final char nextChar = Character.toChars(codePoints[after])[0];
+            if (after < line.length) {
+                final char nextChar = Character.toChars(line[after])[0];
                 if (shouldCheckSeparationFromNextToken(ast, nextChar)
-                        && !Character.isWhitespace(nextChar)) {
+                        && !CommonUtil.isCodePointWhitespace(line, after)) {
```

Comment:
Why change this method call? We have `nextChar` already, why not use it here, too?

Reply:
missed

Reply:
done

Reply:
Resolved.